src/app/pages/account/groups/group-list/group-list.component.ts

```diff
@@ -107,44 +109,46 @@ export class GroupListComponent {
         name: 'delete',
         label : helptext.group_list_actions_label_delete,
         onClick : (members_delete) => {
-          this.entityList.doDelete(members_delete);
+          const self = this;
+          const conf: DialogFormConfiguration = {
+            title: helptext.deleteDialog.title,
+            message: helptext.deleteDialog.message + `<i>${members_delete.group}</i>?`,
+            fieldConfig: [],
+            confirmCheckbox: true,
+            saveButtonText: helptext.deleteDialog.saveButtonText,
+            preInit: function () {
+              if (self.ableToDeleteAllMembers(members_delete)) {
+                conf.fieldConfig.push({
+                  type: 'checkbox',
+                  name: 'delete_users',
+                  placeholder: T(`Delete all ${members_delete.users.length} user(s) with this primary group?`),
```

Comment:
```suggestion
                  placeholder: T(`Delete all ${members_delete.users.length} users with this primary group?`),
```
Parens aren't needed for this plural noun usage.